Coding_Test/TEST/1/Test.java

```diff
@@ -0,0 +1,30 @@
+/*
+  프로그래머스 레벨 2 => N개의 최소 공배수
+ */
+class Solution {
+    public int solution(int[] arr) {
+        int answer = 0;
+        /* 해당 배열의 가장 큰 수 */
+        int max = arr[arr.length - 1];
+        int idx = 1;
+        while (true) {
+          /* max의 배수, idx는 while 문을 돌며 하나씩 ++ */
+            int multiple = max * idx;
+          /* trigger === true => 숫자 n개의 최소 공배수 */
+            boolean trigger = true;
+            for (int num : arr) {
```

Comment:
`for (int i = 0; i < arr.length; i++)`

Reply:
```suggestion
            for (int num : arr) { 
``` 
윗 부분은 ... 반복 .... -> test